packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../common/common';
+
+$defaultColor: $ms-color-neutralTertiary;
```

Comment:
Please prefix these component variables with the component name, to prevent conflicting with other globals. E.g. `$Rating-defaultColor`.